app/src/main/java/com/standuptracker/service/PersonService.kt

```diff
@@ -10,26 +10,27 @@ import retrofit2.Callback
 import retrofit2.Response
 import java.lang.Exception
 
+
 class PersonService {
     fun fetchPeople() : MutableLiveData<ArrayList<Person>> {
-        var _people = MutableLiveData<ArrayList<Person>>()
+        val people = MutableLiveData<ArrayList<Person>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(IPersonDAO::class.java)
         val call = service?.getAllPeople()
         // must put call on the background thread
         call?.enqueue(object: Callback<ArrayList<Person>> {
             override fun onFailure(call: Call<ArrayList<Person>>, t: Throwable) {
-                throw Exception("failed to read JSON")
                 Log.d("JSON", "Save Failed")
+                throw Exception("failed to read JSON")
```

Comment:
If you add code below the throw statement, the it will be unreachable. So it is better if you add it above that line so it can execute.

Reply:
Yes, good point.  This was captured in another code review as well.